utils/data_archive/tests/test_archive_explorer.py

```diff
@@ -11,7 +11,7 @@
 from utils.data_archive.archive_explorer import ArchiveExplorer
 
 
-# pylint:disable=invalid-name
+# pylint:disable=invalid-name,missing-docstring,too-many-public-methods
```

Comment:
Changes to Archive Explorer tests can be remove fro this PR as we are now using ICAT to query this info